### ISSUE TITLE
Sudo not working!

### ISSUE THREAD

**Issue by 0sakib0** [name-is-cipher/boot-nethunter#2]
Using sudo command in normal user returns: sudo effective uid is not 0. Initial boot into kali root works fine but after switching to normal user and then using sudo command returns above response.

**Comment by cipherswami** [comment]
yes, you need to mount fs manually for that in termux and unmount it after use. Since the root user is enabled by default, I ignored it, now that you have mentioned I'll look into it soon when I'm free.

In the meantime, I'm open to any suggestions ...

**Comment by 0sakib0** [comment]
This is a really good project you did cause nethunter terminal is really bugy, keep up the good work thanks.

**Comment by cipherswami** [comment]
Thanks, man. And I found where the issue is. Integrating the solution to boot nethunter might take time, it's a bit complicated. ill tell you the **workaround solution** for now.

You are getting this error because for some reason mounted fs is not accessible by boot nethunter (Termux). when you start chroot in the nethunter app it mounts the fs, but the nethunter app is not giving access to boot nethunter for accessing the mounted fs. In order to use **sudo** in boot nethunter don't start the chroot in the Nethunter app.  And by the way "start chroot" is already integrated into boot nethunter, you don't need to start it in the nethunter app.

**LONG STORY SHORT**:
#> Stop the chroot in nethunter app
#> Clear recent apps 
#> Now start boot nethunter in termux
#=> Now you will be able to use **sudo** for **non-root** users.

**Comment by cipherswami** [comment]
And I found a few more minor issues, ill release a new version with all the fixes after some time.

If you find any more issues or feature requests I'm open.
 And checkout [termux-superuser](https://github.com/name-is-cipher/termux-superuser) it might be helpful since you are into termux.
